src/lib.rs

```diff
@@ -202,10 +189,12 @@ impl Preprocessor for KatexProcessor {
 
     #[tokio::main]
     async fn run(&self, ctx: &PreprocessorContext, mut book: Book) -> Result<Book, Error> {
-        // enforce config requirements
-        enforce_config(&ctx.config);
         // parse TOML config
         let cfg = get_config(&ctx.config)?;
+        if cfg.static_css {
```

Comment:
This is correct because if `static-css = false`, we only need to be called as a preprocessor `[preprocessor.katex]` and there's no need to check for `[output.katex]` or `[output.html]`.